src/stores/pacbioPoolCreate.js

```diff
@@ -916,7 +1032,24 @@ export const usePacbioPoolCreateStore = defineStore('pacbioPoolCreate', {
 
     selectRequest({ id, selected = true }) {
       if (selected) {
-        this.used_aliquots[`_${id}`] = createUsedAliquot({ source_id: id })
+        //If the libraries contain the request, fill the used_aliquot values with the library attributes values for template_prep_kit_box_barcode, volume, concentration, and insert_size
+        const library = Object.values(this.resources.libraries).find(
+          (library) => library.request == id,
+        )
+        const request = this.resources.requests[id]
+        const libraryAttributes = library
```

Comment:
Not entirely sure whats going on here

Reply:
While selecting a request and the request is associated with a library, we are filling the used_aliquot values with the library's attribute values for template_prep_kit_box_barcode, volume, concentration, and insert_size. This is useful for auto-populating selected request information on the pool creation page.
There is a comment on top which describes it. Hope it make sense

Reply:
Ah yes, missed that comment. Might be clearer to manually assign the required attributes instead of filtering the object, not sure.